test/lit.cfg

```diff
@@ -2884,7 +2884,12 @@ def linux_get_lsb_release():
     return ('', '')
 
 if platform.system() == 'Linux':
-    (distributor, release) = linux_get_lsb_release()
+    # Retrieve the Linux distro and version from `/etc/os-release`. 
+    os_release = platform.freedesktop_os_release()
```

Comment:
I don't think we can use `freedesktop_os_release()` :-(

Reply:
Ah that's unfortunate. I replaced this with manual parsing of `/etc/os-release`.